src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -109,7 +109,7 @@ def add_integration_response(self):
 
     def add_permission(self):
         """Add permission to Lambda for the API Trigger."""
-        statement_id = '{}_api_{}'.format(self.app_name, self.trigger_settings['api_name'])
+        statement_id = 'foremast-{}_api_{}'.format(self.app_name, self.trigger_settings['api_name'])
```

Comment:
what if instead of a global prefix, I just add it to all SIDs on permission creation? They pass in the old statement_id and I prefix `foremast` to all indiscriminately?